app/templates/components/bucket-properties.hbs

```diff
@@ -34,18 +34,28 @@
       <td class='key'>Search Index:</td>
       <td class='value'>{{model.props.searchIndexName}}</td>
     </tr>
-    <tr>
-      <td class='key'>Search Schema:</td>
-      <td class='value'>
-        <a href="{{model.props.cluster.clusterProxyUrl}}/search/schema/{{model.props.index.schema}}">
-          {{model.props.index.schema}}
-        </a>
-      </td>
-    </tr>
-    <tr>
-      <td class='key'>Index N_Val:</td>
-      <td class='value'>{{model.props.index.n_val}}</td>
-    </tr>
+    {{#if model.index}}
+      <tr>
+        <td class='key'>Search Schema:</td>
+        <td class='value'>
+          <a href="{{model.cluster.proxyUrl}}/search/schema/{{model.index.schema}}">
```

Comment:
> Since this is appearing in two different templates, maybe we should make this url value a computed property on the model. Just a thought.

Not a bad idea. Let me do that real quick.